### PR TITLE
DNS-SEC support

### PR DESCRIPTION
**Pull request by archaephyrryx**
Added support for TLSA record lookup and AD (authenticated data) bit to query headers

Type signatures of some internal functions (see commit notes) were changed, but this shouldn't break anything as the API is largely unmodified.

### CODE REVIEW

File: Network/DNS/Internal.hs [hunk 1]
```diff
@@ -216,6 +222,7 @@ instance Show RData where
   show (RD_SRV pri wei prt dom) = show pri ++ " " ++ show wei ++ " " ++ show prt ++ BS.unpack dom
   show (RD_OPT od) = show od
   show (RD_OTH is) = show is
+  show (RD_TLSA use sel mtype dgst) = show use ++ " " ++ show sel ++ " " ++ show mtype ++ " " ++ (BS.unpack $ L.toStrict . L.toLazyByteString . L.byteStringHex $ dgst)
```
Comment (archaephyrryx) [inline Network/DNS/Internal.hs]: Display TLSA hash data as hex string rather than raw bytes

**Comment by kazu-yamamoto** [comment]
I have merged this. Thanks.

**Comment by kazu-yamamoto** [comment]
We need to fix a issue of blaze-builder vs bytestring-builder. Logically, only bytestring-builder is necessary.

**Comment by kazu-yamamoto** [comment]
A new version has been released.

**Pull request closed without merging**